MAUI/Sunburst-Chart/Getting-Started.md

```diff
@@ -0,0 +1,395 @@
+                    new SunburstModel { Country = "India", JobDescription = "HR Executives", EmployeesCount = 70 },
+                    new SunburstModel { Country = "India", JobDescription = "Accounts", EmployeesCount = 45 },
+
+                    new SunburstModel { Country = "Germany", JobDescription = "Sales", JobGroup = "Executive", EmployeesCount = 30 },
+                    new SunburstModel { Country = "Germany", JobDescription = "Sales", JobGroup = "Analyst", EmployeesCount = 40 },
+                    new SunburstModel { Country = "Germany", JobDescription = "Marketing", EmployeesCount = 50 },
+                    new SunburstModel { Country = "Germany", JobDescription = "Technical", JobGroup = "Testers", EmployeesCount = 40 },
+                    new SunburstModel { Country = "Germany", JobDescription = "Technical", JobGroup = "Developers", JobRole = "Windows", EmployeesCount = 65 },
+                    new SunburstModel { Country = "Germany", JobDescription = "Technical", JobGroup = "Developers", JobRole = "Web", EmployeesCount = 27 },
+                    new SunburstModel { Country = "Germany", JobDescription = "Management", EmployeesCount = 33 },
+                    new SunburstModel { Country = "Germany", JobDescription = "Accounts", EmployeesCount = 55 },
+
+                    new SunburstModel { Country = "UK", JobDescription = "Technical", JobGroup = "Testers", EmployeesCount = 25 },
+                    new SunburstModel { Country = "UK", JobDescription = "Technical", JobGroup = "Developers", JobRole = "Windows", EmployeesCount = 96 },
+                    new SunburstModel { Country = "UK", JobDescription = "Technical", JobGroup = "Developers", JobRole = "Web", EmployeesCount = 55 },
+                    new SunburstModel { Country = "UK", JobDescription = "HR Executives", EmployeesCount = 60 },
+                    new SunburstModel { Country = "UK", JobDescription = "Accounts", EmployeesCount = 30 }
+                };
+            }    
+    }
+
+{% endhighlight %} 
+
+{% endtabs %} 
+
+Set the `SunburstViewModel` instance as the `BindingContext` of your page to bind `SunburstViewModel` properties to the chart.
+
+N> Add the namespace of `SunburstViewModel` class to your XAML Page, if you prefer to set `BindingContext` in XAML.
+
+{% tabs %} 
+
+{% highlight xaml %} 
+
+    <ContentPage
+        . . .
+        xmlns:sunburst="clr-namespace:Syncfusion.Maui.SunburstChart;assembly=Syncfusion.Maui.SunburstChart"
+        xmlns:model="clr-namespace:SunburstGettingStarted">
+
+    <ContentPage.BindingContext>
+        <model:SunburstViewModel></model:SunburstViewModel>
+    </ContentPage.BindingContext>
+    
+    </ContentPage>
+
+{% endhighlight %}
+
+{% highlight C# %} 
+
+    this.BindingContext = new ViewModel();
+
+{% endhighlight %}
+
+{% endtabs %} 
+
+## Populate chart with data
+
+ Binding `DataSource` to the Sunburst chart [ItemsSource]() property from its BindingContext to create our own Sunburst chart.
```

Comment:
Include the content for adding SunburstHierarchicalLevel and ValueMemberPath

Reply:
content Included